addons/mail/models/mail_blacklist.py

```diff
@@ -44,29 +44,22 @@ def create(self, values):
         emails = [v['email'] for v in new_values]
         self._cr.execute(sql, (emails,))
         bl_entries = dict(self._cr.fetchall())
-        to_create = [v for v in new_values
-                     if v['email'] not in bl_entries]
+        to_create = [v for v in new_values if v['email'] not in bl_entries]
 
         # TODO DBE Fixme : reorder ids according to incoming ids.
         results = super(MailBlackList, self).create(to_create)
         return self.env['mail.blacklist'].browse(bl_entries.values()) | results
 
-    @api.multi
-    def write(self, values):
```

Comment:
Why removing the write override ?

Reply:
It was because edit is not possible anymore through the UI. But anyway, if someone, one day, modifies the blacklist record using the code, it should still normalize the input. As it's not currently the case, do we still have to do this ?